src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -411,10 +412,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('account_name', acct_name_type, id_part=None)
         c.argument('resource_group_name', required=False, validator=process_resource_group)
         c.argument('enable_change_feed', arg_type=get_three_state_flag(), min_api='2019-04-01')
+        c.argument('enable_container_delete_retention',
+                   arg_type=get_three_state_flag(),
```

Comment:
>get_three_state_flag(), [](start = 28, length = 23)

Also add these new arguments to the examples in help.py?

Reply:
currently I don't want to expose in examples because it will need additional registration.